apps/src/templates/SchoolInfoConfirmationDialog.jsx

```diff
@@ -0,0 +1,95 @@
+import React, {Component, PropTypes} from 'react';
+import Dialog, {Body} from '@cdo/apps/templates/Dialog';
+import Button from './Button';
+import SchoolInfoInterstitial from '../lib/ui/SchoolInfoInterstitial';
+import i18n from '@cdo/locale';
+
+export const styles = {
+  button: {
+    marginTop: 30,
+    marginLeft: 290
+  }
+};
+
+class SchoolInfoConfirmationDialog extends Component {
+  static propTypes = {
+    schoolName: PropTypes.string,
+    onUpdate: PropTypes.func,
+    onConfirm: PropTypes.func,
+    isOpen: React.PropTypes.bool.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showSchoolInterstitial: false,
+      schoolName: props.schoolName
+    };
+  }
+
+  handleUpdateClick = () => {
+    this.setState({showSchoolInterstitial: true});
+  };
+
+  renderInitialContent() {
+    const {onConfirm} = this.props;
+    return (
+      <div style={styles.description}>
```

Comment:
It doesn't look like this style exists? (Same with body tag below)

Reply:
Good catch!  removed unused style.